numpy/core/src/multiarray/alloc.c

```diff
@@ -25,10 +25,14 @@
 
 #include <assert.h>
 
-#ifdef HAVE_SYS_MMAN_H
+#ifdef NPY_OS_LINUX
```

Comment:
Is `NPY_OS_LINUX` a true superset of `HAVE_SYS_MMAN_H`? Just curious.

Reply:
`<sys/mman.h>` dates back to Unix and I've confirmed that it exists [at least since kernel 2.6.0](http://tomoyo.osdn.jp/cgi-bin/lxr/source/include/linux/mman.h?v=linux-2.6.0) (possibly before), so I think it's a safe bet.

Reply:
Sorry, I mean that `NPY_OS_SOLARIS` or `NPY_OS_DARWIN` may currently succeed `HAVE_SYS_MMAN_H ` but are not covered under `NPY_OS_LINUX `

Reply:
That's right, but we're only including `<sys/mman.h>` for `MADV_HUGEPAGE`, and, AFAIK, that's a feature only available on Linux.